### DISCUSSION TITLE
yiwen-blog-website/docs/native/ReactNative/expo/intro

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#273] [Announcements]
# yiwen-blog-website/docs/native/ReactNative/expo/intro

文档

https://hao-yiwen.github.io/yiwen-blog-website/docs/native/ReactNative/expo/intro

**Comment by Hao-yiwen** [comment]
Expo在RN的基础上套了一层开发的壳子，这层壳子对开发体验有很大的影响，但是从本质来说，Expo只是资源整合，并没有突破性的新东西。Expo对新手更加友好，对开发经验较为丰富的开发者而言，或许SDK才是最有吸引力的东西。另外抽象配置有好有坏，对前端更友好，对移动端可能难以理解。

**Reply by Hao-yiwen** [reply]
expo在包体积优化和expo module层面有一些突破，在这两方面极大简化了开发者成本。
